tools/spectral/ipa/CONTRIBUTING.md

```diff
@@ -0,0 +1,238 @@
+# Contributing to IPA Validations
+
+Thank you for your interest in contributing! We welcome contributions of all kinds—bug fixes, new features, documentation improvements, and more.
+
+**Note:** For MongoDB engineers, please review https://go/ipa-validation-internal-wiki for additional information.
+
+## 🛠 Getting Started
+
+### Prerequisites
+
+- Node.js >= v20
+- npm
+
+1. **Fork the repo**
+2. **Clone your fork**
+   ```bash
+   git clone https://github.com/your-username/project-name.git
+   ```
+3. **Install dependencies**
+   ```bash
+   cd project-name
+   npm install
+   ```
+4. **Create a new branch**
+   ```bash
+   git checkout -b my-feature-branch
+   ```
```

Comment:
Not relevant. Those are usually for repo root. We have openapi monorepo so I would skip those completely - specifically that those cover genric way of working with git repos